DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/bridging/test_bridging_no_fwd_port_down.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+import asyncio
```

Comment:
And also add your variables in constants.py.
This will be your first PR with bridging tests. So let's have all these files here.